site/components/common/Footer/Footer.tsx

```diff
@@ -9,6 +10,13 @@ import { Logo, Container } from '@components/ui'
 import { I18nWidget } from '@components/common'
 import s from './Footer.module.css'
 
+const ThemeSwitcher = dynamic(() => import('@components/ui/ThemeSwitcher'), {
+  ssr: false,
+  loading: () => (
+    <div className="w-24 h-10 rounded-md bg-accent-2 animate-pulse" />
```

Comment:
This loading state is smaller than the actual input so there's a CLS. For this case I don't see a reason to use lazily load the component, it's small and doesn't load large libraries.

Reply:
I will make the rotation and remove the lazy loading. I did that because the themes array and the current theme come from the client side, so it showed a hydration issue instead of fixing it, showing nothing, decided to show a placeholder until loads and is read.